test/contentful/utils.js

```diff
@@ -16,10 +16,16 @@ const SPACE = {
 };
 const LANGUAGE = 'klingon';
 
-configureContentful({ space: SPACE, language: LANGUAGE, url: '/v2/contentfull_proxy.json' });
-
 
 describe('modules/contentful/utils', () => {
+  beforeEach(() => {
+    configureContentful({
+      space: SPACE,
+      language: LANGUAGE,
+      url: '/v2/contentfull_proxy.json',
```

Comment:
is ist with one L or with two LL ?

Reply:
I think it's only one `l` but don't care too much for tests